### PR TITLE
chore: added links to sections and workflow inputs documentation

### PR DESCRIPTION
**Pull request by drodriguez-305**
### Which problem does the PR fix?
Part of a larger epic focusing on integration tests and improvements of integration tests, docs, etc.
Related to https://github.com/camunda/distribution/issues/241

### What's in this PR?
Adding documentation on integration test inputs for teams trying to configure integration tests on their repositories. 

### Checklist

Please make sure to follow our [Contributing Guide](../blob/main/CONTRIBUTING.md).

**Before opening the PR:**

- [ ] In the repo's root dir, run `make go.update-golden-only`.
- [ ] There is no other open [pull request](../pulls) for the same update/change.
- [ ] Tests for charts are added (if needed).
- [ ] In-repo [documentation](../blob/main/CONTRIBUTING.md#documentation) are updated (if needed).

**After opening the PR:**

- [ ] Did you sign our CLA (Contributor License Agreement)? It will show once you open the PR.
- [ ] Did all checks/tests pass in the PR?

### CODE REVIEW

**Review by jessesimpson36 — APPROVED** [review]

File: docs/gha-workflows.md [hunk 1]
```diff
@@ -145,3 +146,72 @@ jobs:
...
+      # Required: false
+      extra-values: |
+        global:
+          image:
+            tag: 8.2.10
+        console:
+          image:
+            tag: xyz
+
+# Note:
```
Comment (aabouzaid) [inline docs/gha-workflows.md:208]: Let's add this outside the code part (just before `## Additional Information`)

File: docs/gha-workflows.md [hunk 2]
```diff
@@ -145,3 +146,72 @@ jobs:
...
+      camunda-helm-git-ref: 'main'
+
+      # Git reference of the caller's repository (branch, tag, or commit SHA) that initiated the workflow
+      # Default: 'main'
+      # Required: false
+      caller-git-ref: 'feature-branch'
+
+      # Whether to keep the test deployment after the workflow is completed
+      # Default: false
+      # Note: All persistent deployments will be deleted frequently to save costs
```
Comment (aabouzaid) [inline docs/gha-workflows.md:179]: Move the `Note` above the `Default`.

File: docs/gha-workflows.md [hunk 3]
```diff
@@ -145,3 +146,72 @@ jobs:
     with:
       github-run-id: "${{ github.event.workflow_run.id }}"
 ```
+# GitHub Actions: Integration Test Template Configuration
+
+This section details the inputs for the GitHub Actions workflow defined in `.github/workflows/test-integration-template.yaml` within the Camunda Platform Helm repository. These inputs allow you to customize Helm chart deployments for integration testing.
+
+## Workflow Inputs
```
Comment (aabouzaid) [inline docs/gha-workflows.md:153]: Please add a link to this section inside the workflow itself so it's easy to find.
https://github.com/camunda/camunda-platform-helm/blob/6ab109cc68bf5befd225be792cab29598da23dbf/.github/workflows/test-integration-template.yaml#L1

**Review by aabouzaid — CHANGES_REQUESTED** [review]

File: docs/gha-workflows.md (hunk 1 above)
Comment (drodriguez-305) [reply docs/gha-workflows.md]: done

File: docs/gha-workflows.md (hunk 2 above)
Comment (drodriguez-305) [reply docs/gha-workflows.md:178]: done

File: docs/gha-workflows.md (hunk 3 above)
Comment (drodriguez-305) [reply docs/gha-workflows.md:153]: I just fixed the name and directly linked it on there.

**Review by aabouzaid — APPROVED** [review]
Great job @drodriguez-305 :clap: 
I made a small tidy-up, feel free to merge.

**Pull request merged**